From ea24235c40215f821bf779667f28361749f7a448 Mon Sep 17 00:00:00 2001
From: Walter Bright <walter@walterbright.com>
Date: Wed, 30 Oct 2013 19:43:43 -0700
Subject: [PATCH 5/5] add missing files to MANIFEST

---
 mak/MANIFEST | 23 ++++++++++++++++++++++-
 1 file changed, 22 insertions(+), 1 deletion(-)

diff --git a/mak/MANIFEST b/mak/MANIFEST
index 21651571..6c3b2233 100644
--- a/mak/MANIFEST
+++ b/mak/MANIFEST
@@ -242,4 +242,25 @@ MANIFEST=\
 	src\rt\util\string.d \
 	src\rt\util\utf.d \
 	\
-	src\etc\linux\memoryerror.d
+	src\etc\linux\memoryerror.d \
+	\
+	test\init_fini\src\runtime_args.d \
+	test\init_fini\src\thread_join.d \
+	test\init_fini\Makefile \
+	test\shared\src\host.c \
+	test\shared\src\lib.d \
+	test\shared\src\liblinkdep.d \
+	test\shared\src\libloaddep.d \
+	test\shared\src\link.d \
+	test\shared\src\linkD.c \
+	test\shared\src\linkDR.c \
+	test\shared\src\link_linkdep.d \
+	test\shared\src\link_loaddep.d \
+	test\shared\src\load.d \
+	test\shared\src\loadDR.c \
+	test\shared\src\load_linkdep.d \
+	test\shared\src\load_loaddep.d \
+	test\shared\src\plugin.d \
+	test\shared\.gitignore \
+	test\shared\Makefile \
+
-- 
2.34.1

