From e17e774dd390552c524eb12980cad1f8fca7e450 Mon Sep 17 00:00:00 2001
From: Vladimir Panteleev <git@cy.md>
Date: Sat, 9 Jul 2022 13:52:16 +0000
Subject: [PATCH 5/6] dub.sdl: Update paths

---
 dub.sdl | 48 ++++++++++++++++++++++++------------------------
 1 file changed, 24 insertions(+), 24 deletions(-)

diff --git a/dub.sdl b/dub.sdl
index ad7a695f61..af0c561f6c 100644
--- a/dub.sdl
+++ b/dub.sdl
@@ -10,7 +10,7 @@ dependency ":frontend" version="*"
 subPackage {
   name "root"
   targetType "library"
-  sourcePaths "src/dmd/common" "src/dmd/root"
+  sourcePaths "compiler/src/dmd/common" "compiler/src/dmd/root"
 }
 
 subPackage {
@@ -19,16 +19,16 @@ subPackage {
   sourcePaths
 
   sourceFiles \
-    "src/dmd/console.d" \
-    "src/dmd/entity.d" \
-    "src/dmd/errors.d" \
-    "src/dmd/file_manager.d" \
-    "src/dmd/globals.d" \
-    "src/dmd/id.d" \
-    "src/dmd/identifier.d" \
-    "src/dmd/lexer.d" \
-    "src/dmd/tokens.d" \
-    "src/dmd/utils.d"
+    "compiler/src/dmd/console.d" \
+    "compiler/src/dmd/entity.d" \
+    "compiler/src/dmd/errors.d" \
+    "compiler/src/dmd/file_manager.d" \
+    "compiler/src/dmd/globals.d" \
+    "compiler/src/dmd/id.d" \
+    "compiler/src/dmd/identifier.d" \
+    "compiler/src/dmd/lexer.d" \
+    "compiler/src/dmd/tokens.d" \
+    "compiler/src/dmd/utils.d"
 
   versions \
     "CallbackAPI" \
@@ -47,7 +47,7 @@ subPackage {
   preGenerateCommands `"%DUB_EXE%" --arch=%DUB_ARCH% --compiler="%DC%" --single "%DUB_PACKAGE_DIR%config.d" -- "%DUB_PACKAGE_DIR%generated/dub" "%DUB_PACKAGE_DIR%VERSION"` platform="windows"
 
   stringImportPaths \
-    "src/dmd/res" \
+    "compiler/src/dmd/res" \
     "generated/dub"
 
   dependency "dmd:root" version="*"
@@ -59,11 +59,11 @@ subPackage {
   sourcePaths
 
   sourceFiles \
-    "src/dmd/astbase.d" \
-    "src/dmd/parse.d" \
-    "src/dmd/transitivevisitor.d" \
-    "src/dmd/permissivevisitor.d" \
-    "src/dmd/strictvisitor.d"
+    "compiler/src/dmd/astbase.d" \
+    "compiler/src/dmd/parse.d" \
+    "compiler/src/dmd/transitivevisitor.d" \
+    "compiler/src/dmd/permissivevisitor.d" \
+    "compiler/src/dmd/strictvisitor.d"
 
   versions "CallbackAPI"
 
@@ -73,8 +73,8 @@ subPackage {
 subPackage {
   name "frontend"
   targetType "library"
-  sourcePaths "src/dmd"
-  stringImportPaths "src/dmd/res"
+  sourcePaths "compiler/src/dmd"
+  stringImportPaths "compiler/src/dmd/res"
 
   versions \
     "NoBackend" \
@@ -83,10 +83,10 @@ subPackage {
     "MARS" \
     "CallbackAPI"
 
-  excludedSourceFiles "src/dmd/backend/*"
-  excludedSourceFiles "src/dmd/root/*"
-  excludedSourceFiles "src/dmd/common/*"
-  excludedSourceFiles "src/dmd/{\
+  excludedSourceFiles "compiler/src/dmd/backend/*"
+  excludedSourceFiles "compiler/src/dmd/root/*"
+  excludedSourceFiles "compiler/src/dmd/common/*"
+  excludedSourceFiles "compiler/src/dmd/{\
     astbase,\
     console,\
     entity,\
@@ -104,7 +104,7 @@ subPackage {
     utf,\
     utils\
   }.d"
-  excludedSourceFiles "src/dmd/{\
+  excludedSourceFiles "compiler/src/dmd/{\
     dmsc,\
     e2ir,\
     eh,\
-- 
2.36.1

